src/Pos/Wallet/WalletMode.hs

```diff
@@ -190,10 +196,16 @@ type WalletMode ssc m
 -- Implementations of 'WalletMode'
 ---------------------------------------------------------------
 
-type WalletRealMode = PeerStateHolder
-                      (KeyStorage
-                       (WalletDB
+type FakeSsc ssc = TaggedTrans ssc IdentityT
```

Comment:
~it's one more reason for your butthurt :peka:~
this transformer needed to implement `WithNodeContext` for lite-wallet to satisfy constraints from full-wallet (because lite-wallet is absolute bardaq)